server/src/test/java/io/crate/integrationtests/RoleManagementIntegrationTest.java

```diff
@@ -277,4 +277,14 @@ public void testDropSuperUserThrowsException() {
             .hasHTTPError(BAD_REQUEST, 4004)
             .hasMessageContaining("Cannot drop a superuser 'crate'");
     }
+
+    @Test
+    public void test_create_user_jwt_properties_must_be_unique() {
```

Comment:
Can we have a test with different jwt properties, but with the same username (so user itself already exists, and not clash with existing jwt properties).

Reply:
I pushed only unit test. Tested locally that such scenario shows proper message
`Role  user1 or another role with the same combination of jwt properties already exists`